#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_interactive_marker'

try:
    import imp
    imp.find_module(PACKAGE)
    from dynamic_reconfigure.parameter_generator_catkin import *;
except:
    import roslib; roslib.load_manifest(PACKAGE)
    from dynamic_reconfigure.parameter_generator import *;

from math import pi

gen = ParameterGenerator ()
gen.add("point_size", double_t, 0, "size of interactive marker points", 0.002, 0, 1.0)

exit (gen.generate (PACKAGE, "jsk_interactive_marker", "InteractivePointCloud"))
